### PR TITLE
fixed landscape screen issue with intro screen

### PR DESCRIPTION
**Pull request by criticalAY**
## Pull Request template
FIXME : locked orientation issue

## Purpose / Description
allows horizontal screen on intro screen

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?
Tested on oneplus nord ce

https://user-images.githubusercontent.com/48384865/211105290-3f0879d7-561b-43fc-ad4a-e8a165bf4806.mp4

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
I particularly don't like it being scrollable, as everything could fit in the screen

**Comment by criticalAY** [comment]
I have done few changes and I followed the best practices suggested by android/google docs and their won't be an issue with the bugs as the file names are same and we are not handling the changes manually android itself will take care of it. [Orientation qualifiers](https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes#:~:text=Even%20though%20you%20may%20be%20able%20to%20support%20all%20size%20variations%20using%20only%20combinations%20of%20the%20smallest%20width%20and%20available%20width%20qualifiers%2C%20you%20might%20also%20want%20to%20change%20the%20user%20experience%20when%20the%20user%20switches%20between%20portrait%20and%20landscape%20orientations.) This is unlike the code snippet I implement and sent on discord.

**Comment by criticalAY** [comment]
Brayan here is the latest updated test file

https://user-images.githubusercontent.com/48384865/211143394-f0664751-f7dd-43ea-988c-8f4713e49145.mp4

File: AnkiDroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -410,13 +410,12 @@
...
             />
-        <!-- TODO: Fix LockedOrientationActivity - introduction_layout.xml -->
+
         <activity android:name=".IntroductionActivity"
             android:exported="false"
             android:theme="@style/Theme.AppCompat.NoActionBar"
-            android:screenOrientation="userPortrait"
-            android:configChanges="keyboardHidden|orientation|screenSize|locale"
-            tools:ignore="LockedOrientationActivity" />
+            android:screenOrientation="user"
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/AndroidManifest.xml:417]: Can we remove this line now?

**Review by BrayanDSO — APPROVED** [review]
Very good 👏👏👏. Left only a nit

File: AnkiDroid/src/main/AndroidManifest.xml (hunk 1 above)
Comment (criticalAY) [reply AnkiDroid/src/main/AndroidManifest.xml]: yes we can

**Review by pratyaksh1610 — APPROVED** [review]

**Review by david-allison — CHANGES_REQUESTED** [review]
Sorry to nitpick, but it's the first screen users will see

UI: Needs a little more spacing between 'strongest' and 'get started'

(I'd say ~8dp, but do what you think looks good)

**Comment by criticalAY** [comment]
> Sorry to nitpick, but it's the first screen users will see
> 
> UI: Needs a little more spacing between 'strongest' and 'get started'
> 
> (I'd say ~8dp, but do what you think looks good)

Agreed looks better so made a commit.

**Comment by BrayanDSO** [comment]
Can we make the buttons aligned horizontally then? I think it will look better this way

**Comment by criticalAY** [comment]
> Can we make the buttons aligned horizontally then (on landscape mode)? I think it will look better this way

Umm yes a lot of negative space can be avoided by this way

**Comment by criticalAY** [comment]
test3
Here is the latest test file, Thanks Brayan I was keep the original format of buttons but this one is better considering the space we have. I think we are done with the nits now :)

**Comment by BrayanDSO** [comment]
@david-allison do we need the `Done` button? I guess that the answer would be "yes", but I don't know if this is noted somewhere else.

**Comment by david-allison** [comment]
'Done' is a default of the AppIntro library. I lean towards 'remove'.

**Review by david-allison — APPROVED** [review]

**Pull request merged**